.github/workflows/godwoken-tests.yml

```diff
@@ -10,7 +10,8 @@ on:
 
 jobs:
   godwoken-tests:
-    uses: nervosnetwork/godwoken-tests/.github/workflows/reusable-integration-test-v1.yml@develop
+    uses: classicalliu/godwoken-tests/.github/workflows/reusable-integration-test-v1.yml@add-kicker-repository-input
     with:
       web3_ref: "${{ github.sha }}"
-      kicker_ref: "compatibility-changes"
+      kicker_repository: "classicalliu/godwoken-kicker"
+      kicker_ref: "for-refactor-db"
```

Comment:
If you use this kind of ref ([`refs/pull/253/head`](https://github.com/RetricSu/godwoken-kicker/pull/253)), then we don't need to change the `kicker_repository`.

Reply:
I see, updated